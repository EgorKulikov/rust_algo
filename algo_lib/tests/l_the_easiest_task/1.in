3 2
3 1 2 3
3 1 2 4
0
1 2 2
2 3 3
