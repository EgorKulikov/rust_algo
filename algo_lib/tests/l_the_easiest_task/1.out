1 1
3 1 2 2
2 3 4
